src/send/test.rs

```diff
@@ -95,7 +107,7 @@ fn without_optional_data() {
 fn missing_to_field() {
     let client = setup();
 
-    let response = client
+    let mut response = client
```

Comment:
Curious why this was changed to be mutable? The code below doesn't appear to be changing the value?

Reply:
It's because `Response::body` is `&mut self`, it returns a mutable reference to the body.

Ref: https://api.rocket.rs/rocket/response/struct.Response.html#method.body